core/src/main/java/feast/core/service/SpecService.java

```diff
@@ -134,11 +137,12 @@ public GetFeatureSetResponse getFeatureSet(GetFeatureSetRequest request)
 
   /**
    * Return a list of feature sets matching the feature set name, version, and project provided in
-   * the filter. All fields are requried. Use '*' for all three arguments in order to return all
-   * feature sets and versions in all projects.
+   * the filter. Use '*' for all three arguments in order to return all feature sets and versions in
+   * all projects.
```

Comment:
Shouldn't this be limited to the default project?

Reply:
It will default to the default project if its omitted.